controller/controller_server/csi_controller_server.py

```diff
@@ -563,21 +562,21 @@ def GetPluginInfo(self, request, context):
         logger.info("finished GetPluginInfo")
         return csi_pb2.GetPluginInfoResponse(name=name, vendor_version=version)
 
-    def _get_volume_name_and_prefix(self, request, array_mediator):
-        return self._get_object_name_and_prefix(request, array_mediator.max_volume_prefix_length,
-                                                array_mediator.max_volume_name_length,
-                                                config.OBJECT_TYPE_NAME_VOLUME,
-                                                config.PARAMETERS_VOLUME_NAME_PREFIX)
+    def _get_volume_name(self, request, array_mediator):
+        return self._get_object_full_name(request, array_mediator.max_volume_prefix_length,
+                                          array_mediator.max_volume_name_length,
+                                          config.OBJECT_TYPE_NAME_VOLUME,
+                                          config.PARAMETERS_VOLUME_NAME_PREFIX)
 
     def _get_snapshot_name(self, request, array_mediator):
-        name, _ = self._get_object_name_and_prefix(request, array_mediator.max_snapshot_prefix_length,
-                                                   array_mediator.max_snapshot_name_length,
-                                                   config.OBJECT_TYPE_NAME_SNAPSHOT,
-                                                   config.PARAMETERS_SNAPSHOT_NAME_PREFIX)
+        name = self._get_object_full_name(request, array_mediator.max_snapshot_prefix_length,
+                                          array_mediator.max_snapshot_name_length,
+                                          config.OBJECT_TYPE_NAME_SNAPSHOT,
+                                          config.PARAMETERS_SNAPSHOT_NAME_PREFIX)
         return name
 
-    def _get_object_name_and_prefix(self, request, max_name_prefix_length, max_name_length, object_type,
-                                    prefix_param_name):
+    def _get_object_full_name(self, request, max_name_prefix_length, max_name_length, object_type,
```

Comment:
align related `get_x_name` function names, IMO by removing `full_`

Reply:
actually, does "get" is the correct term here? its more like "create", "append" or "format", we connect the prefix and name, we don't get it as is.

Reply:
if adding a prefix is all it did, then we could've changed it to something more specific, like `join`.
but since it can also hash (and validates max lengths, etc...) then I don't know which verb can be more accurate.
but as I [wrote](https://github.com/IBM/ibm-block-csi-driver/pull/199/files#r450185142), perhaps something like `get_final_name`/`get_name_in_storage` can solve it

Reply:
done